### ISSUE TITLE
For discussion: use case for power line frequency field

### ISSUE THREAD

**Issue by asoskic** [INCF/artem-is#81]
Guiomar has proposed that it may make sense to add information about the utility frequency at the place where a study has been conducted. We are wondering whether there may be a scenario where such information is useful and the person needing it cannot extract it either from the paper (participant sample description) or from the data that they are reanalysing (frequency spectrum) - it would be good to add questions that someone may make use of, but it is also important not to burden users with unnecessary questions.

**Comment by robertoostenveld** [comment]
`PowerLineFrequency` is documented in BIDS datasets as a required field. 

The frequency of the power line is either 50 or 60 Hz, and from information in the paper it might be deduced from the specification where the lab is located (although not trivial in the case of Japan, where 50 and 60 are both used). It is not a piece of information commonly specified in papers, and as reader/reviewer I am not missing it. If there are potential concerns with power-line affecting the interpretation (e.g. when studying gamma-band activity), then that is usually explicitly discussed.

To summarize: I am neutral to this.

**Comment by sanchalitorpe-source** [comment]
Thanks for the discussion. Power line frequency is already documented as a required field in BIDS datasets. While it can be relevant in specific cases (e.g., gamma-band analyses), it is usually deducible from the data or study context and rarely reported explicitly in papers.

Given the limited added value and the desire to avoid unnecessary user burden, this is left as a “for later” consideration.
